br.app.pw3270.yml

```diff
@@ -0,0 +1,53 @@
+app-id: br.app.pw3270
+runtime: org.gnome.Platform
+sdk: org.gnome.Sdk
+command: pw3270
+finish-args:
+  - "--share=network"
+  - "--share=ipc"
+  - "--socket=fallback-x11"
+  - "--socket=wayland"
+  - "--own-name=br.app.pw3270"
```

Comment:
the application's own-name is granted based on the app-id. Also it is subject to the same requirements as the app-id.